From 39c029cb07078f1cdb87d3cdf61c7da90c802916 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:30:45 -0400
Subject: [alsa] lx_core: Fix dev_dbg typo

Message-id: <140972823573-273-patchset-jkysela@redhat.com>
Patchwork-id: 91140
O-Subject: [RHEL 7.1 PATCH 272/442] ALSA: lx_core: Fix dev_dbg typo
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

Commit be4e6d3c0fa0 ("ALSA: lx6464es: Use standard printk helpers") converted
the custom printk helpers that were used before to standard dev_* functions.
One of the dev_dbg calls had a typo, that was hidden away by an #if 0 .. #endif

Signed-off-by: Maxime Ripard <maxime.ripard@free-electrons.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>
Signed-off-by: Jaroslav Kysela <perex@perex.cz>

(picked from kernel: 68e440bb48a35c65756471e3cb952ed7fb7cef5b)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/lx6464es/lx_core.c b/sound/pci/lx6464es/lx_core.c
index 652f6df..9e0acba 100644
--- a/sound/pci/lx6464es/lx_core.c
+++ b/sound/pci/lx6464es/lx_core.c
@@ -1197,7 +1197,7 @@ irqreturn_t lx_interrupt(int irq, void *dev_id)
 
 #if 0
  if (irqsrc & MASK_SYS_STATUS_EOBI)
-  dev_dgg(chip->card->dev, "interrupt: EOBI\n");
+  dev_dbg(chip->card->dev, "interrupt: EOBI\n");
 
  if (irqsrc & MASK_SYS_STATUS_EOBO)
   dev_dbg(chip->card->dev, "interrupt: EOBO\n");
-- 
1.7.1